app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -918,3 +918,27 @@ export const appendSignedMessage = async (ethereumClientService: EthereumClientS
 		signedMessages: simulationState.signedMessages.concat(signedMessage)
 	}
 }
+
+// takes the most recent block that the application is querying and does the calculation based on that
+export const getSimulatedFeeHistory = async (ethereumClientService: EthereumClientService, simulationState: SimulationState | undefined, request: FeeHistory): Promise<EthGetFeeHistoryResponse> => {
+	//const numberOfBlocks = request.params[0]
+	const blockTag = request.params[1]
+	const rewardPercentiles = request.params[2]
+	const newestBlock = await getSimulatedBlock(ethereumClientService, simulationState, blockTag, true)
+	const blocksToConsider = [newestBlock]
+	const oldestBlock = blocksToConsider[0]?.number
+	if (oldestBlock === undefined) throw new Error('oldest block was undefined')
```

Comment:
`blocksToConsider[0]` will always be `newestBlock`, why not just use that directly?

Reply:
I wanted to add a support for multiple blocks, in future